plenum/common/messages/fields.py

```diff
@@ -376,6 +376,25 @@ def _specific_validation(self, val):
             return 'empty serialized value'
 
 
+class VersionField(FieldBase):
+    _base_types = (str,)
+
+    def __init__(self, comp_num=(2, 3,), **kwargs):
```

Comment:
Not sure about default value `(2, 3,)`. It makes sense set `comp_num` more explicit way in place there a instance of the class is created.